spring-ai-core/src/test/java/org/springframework/ai/parser/BeanOutputParserTest.java

```diff
@@ -100,6 +122,30 @@ public void shouldReturnFormatContainingJsonSchemaIncludingPropertyAndPropertyDe
 					""");
 		}
 
+		@Test
+		public void usesLineEndingsNormalizer() {
```

Comment:
Test case methods in JUnit 5 can be package-private now (recommended).

Reply:
Thanks John! I'll change them. I originally had it public just to match the other tests in BeanOutputParserTest.